optics-core/src/Optics/Fold.hs

```diff
@@ -395,6 +429,7 @@ sumOf o = foldlOf' o (+) 0
 {-# INLINE sumOf #-}
 
 -- | The sum of a collection of actions, generalizing 'concatOf'.
+-- TODO: we don't yet have 'concatOf'!
```

Comment:
I didn't add `concatOf` because `concatOf` is a restricted version of `foldOf` (though the comment slipped through).

Reply:
Fixed by removing the reference.